shared/src/hooks/web3Context.tsx

```diff
@@ -1,13 +1,18 @@
-import { BaseProvider } from "@ethersproject/providers";
-import { ethers } from "ethers";
 import React, { ReactElement, useContext } from "react";
-import { getNodeURI } from "../utils/env";
+import { ethers } from "ethers";
+import { BaseProvider } from "@ethersproject/providers";
+import { CHAINID, NODE_URI } from "../utils/env";
 
 export type Web3ContextData = {
   provider: BaseProvider;
 };
 
-const defaultProvider = ethers.getDefaultProvider(getNodeURI());
+// TODO: Fix this in the future
+// Right now we just default to ETH mainnet for the unconnected wallet state
+// This means that we do not show the AVAX vault's details on the app
+const defaultProvider = ethers.getDefaultProvider(
+  NODE_URI[CHAINID.ETH_MAINNET]
```

Comment:
What about pulling this from `window.ethereum.chainId` and default to eth mainnet if it doesn't exist?

```
const defaultProvider = ethers.getDefaultProvider(
  NODE_URI[CHAINID.ETH_MAINNET]
);
```